README.md

```diff
@@ -573,10 +573,12 @@ Native,UI/UX and cool Designing stuff ![Free](https://img.shields.io/badge/-Free
 
 - [Codepen Challenges](https://codepen.io/challenges/) - Each month has a theme and every week there is a new challenge prompt that you can use to build a project online using HTML, CSS and JavaScript. The best projects are featured on Codepen's homepage. ![Free](https://img.shields.io/badge/-Free-brightgreen)
 - [Codewars](https://www.codewars.com/) - Improve your skills by training with others on real code challenges. ![Free](https://img.shields.io/badge/-Free-brightgreen)
+- [CSS Battle](https://cssbattle.dev/) - CSSBattle is an online CSS [Code Golfing](https://en.wikipedia.org/wiki/Code_golf) game. Here, players from all around the world try to visually replicate **Targets** in smallest possible CSS code and battle it out to get to the top of the leaderboard. ![Free](https://img.shields.io/badge/-Free-brightgreen)
 - [CSS Diner](http://flukeout.github.io/) - It's a fun game to learn and practice CSS selectors. ![Free](https://img.shields.io/badge/-Free-brightgreen)
 - [CSS Zen Garden](http://www.csszengarden.com/) - A demonstration of what can be accomplished through CSS-based design, and a chance to do your own. ![Free](https://img.shields.io/badge/-Free-brightgreen)
 - [Flexbox Defense](http://www.flexboxdefense.com) - It is a game that covers the flex properties align-items, justify-content, flex-direction, align-self and has 12 different levels. ![Free](https://img.shields.io/badge/-Free-brightgreen)
 - [Flexbox Froggy](https://flexboxfroggy.com) - This game is also about Flexbox and it covers even more flex properties: align-items, justify-content, align-content, flex-direction, align-self, flex-wrap, flex-flow and has 24 different levels. ![Free](https://img.shields.io/badge/-Free-brightgreen)
+- [Frontend Mentor](https://www.frontendmentor.io/solutions) - Frontend Mentor challenges help you improve your coding skills by building realistic projects. It is excellent for those who want to revisit their frontend knowledge. You will get a huge community if you get stucked in the challenges. ![Free](https://img.shields.io/badge/-Free-brightgreen)
```

Comment:
Please also add a paid badge on this, as front-end mentor also has paid problems.

Reply:
I've added the paid badge. Please review.